app/controllers/ncr/work_orders_controller.rb

```diff
@@ -15,7 +15,7 @@ def create
 
     def edit
       if self.proposal.approved?
-        flash[:warning] = "You are about to modify a fully approved request. Changes will be logged and sent to approvers, and this request may require re-approval, depending on the change."
+        flash.now[:warning] = "You are about to modify a fully approved request. Changes will be logged and sent to approvers, and this request may require re-approval, depending on the change."
```

Comment:
that might be the fix in those cases, but this particular story's scope is just this one.